test/Controller/SubjectControllerTest.php

```diff
@@ -43,6 +43,9 @@ public function it_has_metadata()
         $this->assertSame('https://www.example.com/iiif/ban%2Fner/full/full/0/default.jpg', $crawler->filter('meta[property="og:image"]')->attr('content'));
         $this->assertSame('1800', $crawler->filter('meta[property="og:image:width"]')->attr('content'));
         $this->assertSame('1600', $crawler->filter('meta[property="og:image:height"]')->attr('content'));
+        $this->assertSame('application/rss+xml', $crawler->filter('link[rel="alternate"]')->attr('type'));
+        $this->assertSame('eLife Sciences Subject RSS feed', $crawler->filter('link[rel="alternate"]')->attr('title'));
+        $this->assertSame('/rss/recent-by-subject.xml?subject=subject', $crawler->filter('link[rel="alternate"]')->attr('href'));
```

Comment:
@Maelplaine can we agree on the `title` and `href` values?

Reply:
@nlisgo I think this description would look better in an RSS feed reader: 
eLife Sciences Biochemistry recent articles

Reply:
This should be done now.